backend/src/controllers/loginController.js

```diff
@@ -0,0 +1,82 @@
+import users from "../database/usersDb.js";
+import idGenerator from "../../unique-id.js";
+import fs from "fs";
+import { hashSync, compareSync } from "bcrypt";
+import jwt from "jsonwebtoken";
+import dotenv from "dotenv";
+dotenv.config();
+import { fileURLToPath } from "url";
+import path from "path";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+const filePath = path.join(__dirname, "../../../frontend/views/login.html");
+
+const accessKey = process.env.ACCESS_TOKEN_SECRET;
+const uniqueId = idGenerator();
```

Comment:
Nice idea but you can actually use uuid instead of reinventing the wheel by creating your own uniqued id. UUID is well tested and works well. It is used in most programming languages and there is an npm package for it.